CDBTo3DTiles/src/TileFormatIO.cpp

```diff
@@ -74,9 +75,17 @@ void writeToTilesetJson(const CDBTileset &tileset, bool replace, std::ofstream &
         tilesetJson["root"]["refine"] = "ADD";
     }
 
+    if(threeDTilesNext)
+    {
+      tilesetJson["extensionsUsed"] = nlohmann::json::array();
```

Comment:
You can add the value in the constructor:

```suggestion
      tilesetJson["extensionsUsed"] = nlohmann::json::array({ "3DTILES_implicit_tiling" });
```